server/service/GenericResponseProfileService.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+var fileOperation = require('../node_modules/onf-core-model-ap/applicationPattern/databaseDriver/JSONDriver');
+const onfPaths = require('onf-core-model-ap/applicationPattern/onfModel/constants/OnfPaths');
+
+/**
+ * Returns the Datatype of the Field
+ *
+ * uuid String 
+ * returns inline_response_200_14
+ **/
+exports.getGenericResponseProfileDatatype = function(url) {
+  return new Promise(async function(resolve, reject) {
+    var examples = {};
+    var value = await fileOperation.readFromDatabaseAsync(url);
+    examples['application/json'] = {
+  "response-profile-1-0:datatype" : value
+};
+    if (Object.keys(examples).length > 0) {
+      resolve(examples[Object.keys(examples)[0]]);
+    } else {
+      resolve();
+    }
+  });
+}
+
+
+/**
+ * Returns the Description of the Field
+ *
+ * uuid String 
+ * returns inline_response_200_13
+ **/
+exports.getGenericResponseProfileDescription = function(url) {
+  return new Promise(async function(resolve, reject) {
+    var examples = {};
+    console.log("function",url)
+    var value = await fileOperation.readFromDatabaseAsync(url);
+    console.log(value)
+    examples['application/json'] = {
+  "response-profile-1-0:description" : value
+};
+    if (Object.keys(examples).length > 0) {
+      resolve(examples[Object.keys(examples)[0]]);
+    } else {
+      resolve();
+    }
+  });
+}
+
+
+/**
+ * Returns the name of the Field
+ *
+ * uuid String 
+ * returns inline_response_200_12
+ **/
+exports.getGenericResponseProfileFieldName = function(url) {
+  return new Promise(async function(resolve, reject) {
+    var examples = {};
+    console.log(url)
+    var value = await fileOperation.readFromDatabaseAsync(url);
+    console.log(value)
+    examples['application/json'] = {
+  "response-profile-1-0:field-name" : value
+};
+    if (Object.keys(examples).length > 0) {
+      resolve(examples[Object.keys(examples)[0]]);
+    } else {
+      resolve();
+    }
+  });
+}
+
+
+/**
+ * Returns the name of the Operation
+ *
+ * uuid String 
+ * returns inline_response_200_11
+ **/
+exports.getGenericResponseProfileOperationName = function(url) {
+  return new Promise(async function(resolve, reject) {
+    var examples = {};
+    var value = await fileOperation.readFromDatabaseAsync(url);
+    console.log(value)
+    examples['application/json'] = {
+  "response-profile-1-0:operation-name" : value
+};
+    if (Object.keys(examples).length > 0) {
+      resolve(examples[Object.keys(examples)[0]]);
+    } else {
+      resolve();
+    }
+  });
+}
+
+
+/**
+ * Returns the Value of the Field
+ *
+ * uuid String 
+ * returns inline_response_200_15
+ **/
+exports.getGenericResponseProfileValue = function(url) {
+  return new Promise(async function(resolve, reject) {
+    var examples = {};
```

Comment:
Please change the name examples to response everywhere used , it would be more appropriate naming convention to be used